### PR TITLE
news

### PR DESCRIPTION
**Pull request by Sgemini**
@newStart99

### CODE REVIEW

File: app/assets/javascripts/news.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```
Comment (newStart99) [inline app/assets/javascripts/news.js]: JS is written here?

File: app/assets/stylesheets/news.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the news controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (newStart99) [inline app/assets/stylesheets/news.scss]: This CSS file is dry.

File: app/assets/javascripts/news.js (hunk 1 above)
Comment (Sgemini) [inline app/assets/javascripts/news.js]: Temporarily not, i would refer to my company's project, when i have a conclusion, i will tell you

File: app/assets/stylesheets/news.scss (hunk 2 above)
Comment (Sgemini) [inline app/assets/stylesheets/news.scss]: this file is generate by system

**Comment by newStart99** [comment]
Ok, I understand

**Pull request merged**